<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить данные</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/main.png</normaloff>icons/main.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>331</width>
     <height>186</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="name">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="login">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="url">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="group">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>68</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Название:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string>URL:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="text">
       <string>Группа:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="notes">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>331</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>176</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="ok">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="passwordGeneratorButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>104</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>•••</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>66</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Заметки:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
